src/CommandingOfficers/RuinedCommander.java

```diff
@@ -139,11 +139,11 @@ public int calculateCombatCharge(UnitDelta minion, UnitDelta enemy, boolean isCo
     if( !isInZone(chargeSource) )
       return 0;
 
-    double myHPDealt = enemy.getHPDamage();
+    int guiHPDealt = enemy.getHPDamage() / 10;
```

Comment:
Already in HP so why divide, also maaaagic.

Reply:
Because it's in [0,100]